xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -575,11 +580,7 @@ bool CGUIDialogPVRTimerSettings::Validate()
   // Begin and end time
   if (!bStartAnyTime && !bEndAnyTime)
   {
-    // Not in the set of having neither or both of start clock entry and
-    // end clock entry while also being a timer rule
-    if (!(m_timerType->SupportsStartTime() == m_timerType->SupportsEndTime() &&
```

Comment:
I'd say instead of removing it we need to fix this line. We need to check whether start AND end time is support by the timer type and only in this case do the compare of the time values.

Reply:
Yes, but you guys requested a minimal change ;-) But agreed, we should fix it as well. Updated PR accordingly.